dbcon/mysql/ha_mcs_pushdown.cpp

```diff
@@ -843,9 +813,19 @@ create_columnstore_select_handler(THD* thd, SELECT_LEX* select_lex)
 #endif
             join->conds = conds;
         }
+
+        // MCOL-3747 IN-TO-EXISTS rewrite inside MDB didn't add
+        // an equi-JOIN condition.
+        if (!unsupported_feature && in_subselect_rewrite(select_lex))
+        {
+            unsupported_feature = true;
+            handler->err_msg.assign("create_columnstore_select_handler(): \
+                Internal error occured in optimize_unflattened_subqueries_mcs()");
```

Comment:
in_subselect_rewrite instead of optimize_unflattened_subqueries_mcs

Reply:
+1